vyper/cli/vyper_json.py

```diff
@@ -206,6 +207,19 @@ def get_inputs(input_dict: dict) -> dict[PurePath, Any]:
     return ret
 
 
+def get_storage_layout_overrides(input_dict: dict) -> dict[PurePath, StorageLayout]:
+    storage_layout_overrides: dict[PurePath, StorageLayout] = {}
+
+    for path, value in input_dict.get("storage_layout_overrides", {}).items():
+        if path not in input_dict["sources"]:
+            raise JSONError(f"unknown target for storage layout override: {path}")
```

Comment:
no coverage for this line https://github.com/vyperlang/vyper/pull/4370/checks?check_run_id=34325903794